awx/ui_next/src/screens/Inventory/InventoryGroupHosts/InventoryGroupHostList.jsx

```diff
@@ -67,6 +69,30 @@ function InventoryGroupHostList({ i18n }) {
   };
 
   const isAllSelected = selected.length > 0 && selected.length === hosts.length;
+
+  const {
+    isLoading: isDisassociateLoading,
```

Comment:
should we set this to a variable, or should we setState of the root isLoading and continue to just use that to pass to hasContentLoading?  (question...not sure which would be better)